app/controllers/reviews_controller.rb

```diff
@@ -26,9 +26,10 @@ def create
 
   def lock_controls
     unlocked = @component.rules.where(locked: false)
+    filtered_unlocked = unlocked.where(status: 'Not Yet Determined').as_json.filter { |u| u['satisfied_by'].nil? }
```

Comment:
```suggestion
    filtered_unlocked = unlocked.as_json.filter { |u| u['status'] == 'Not Yet Determined' }
```

Reply:
The satisfied_by isn't nil it's just empty so this wasn't working. The status is correct for duplicates after as_json because it's status is linked to it's satisfied by control.

Reply:
I had to rework the logic a bit differently to get it to work, but here's what I did: https://github.com/mitre/vulcan/pull/405/commits/1c97165aee52e56a75009eae3c13a2afd91d3d45

The changes should be deployed soon